netty/src/main/java/io/grpc/netty/NettyServerStream.java

```diff
@@ -53,6 +58,13 @@
   private final NettyServerHandler handler;
   private final Http2Stream http2Stream;
   private final WriteQueue writeQueue;
+  private final Attributes attributes;
+
+  private static Attributes.Key<SocketAddress> REMOTE_ADDR_KEY =
+      Attributes.Key.named("remote-addr");
```

Comment:
`Key` uses identity for equality (and hash code). So, to be useful, these must be public. And they would likely be shared across transport implementations (e.g. netty and okio can use same attribute keys for these things).

@ejona86, thoughts on where public definitions for these belong?